The Bootstrap Project

ElijahTobs opened the pull request:
Newsweek clone

- This project is meant to test how well the student has mastered bootstrap
- The project was built mostly with bootstrap
- I made use of col, row, justify-content, align-items, container, container-fluid, px, and so many other bootstrap components.
- It is a replica of newsweek.com

ElijahTobs committed:
add new branch and empty files

ElijahTobs committed:
add header

ElijahTobs committed:
add and style navbar

ElijahTobs committed:
Finish styling header and nav

- add margin and padding to nav
- make weather an 'a' tag
- add border-bottom to nav and styled it

ElijahTobs committed:
add headers with navbars

ElijahTobs committed:
Add sections with contents

- add top-story section and style it
- add featured-stories section and style it
- add opinion asection and style it
- add more-stories section and style it
- add culture and travel section and style it
- add heroes-of-oandemic section and style it

ElijahTobs committed:
Add insight sections

ElijahTobs committed:
center second-nav

ElijahTobs committed:
add magazine section

ElijahTobs committed:
add footer top

ElijahTobs committed:
add some part of footer

ElijahTobs committed:
finish styling footer top

ElijahTobs committed:
complete second part of footer

ElijahTobs committed:
update README.md

ElijahTobs committed:
Fix errors

- fix stylelint errors
- fix html validation errors

ElijahTobs committed:
delete unnecessary comments

ElijahTobs committed:
add description to README.md

christianotieno reviewed (approved):
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks:  Keep it up.
##
### The [clone](https://raw.githack.com/ElijahTobs/Newsweek-clone/project/index.html) shows good understanding and usage of building with bootstrap. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the major required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job :+1:

### Pending Changes before merge:
- - [x] Update the README file to include instructions on how to well clone and use the project. Also, the screenshot is not showing, kindly work on that before proceeding.
- - [x] Good job with the summary. We can always improve on it next time by highlighting what we have carried out in the project's PR to guide the reviewer on what has been achieved so far. 

- Consider this complete. You don't have to submit another CRR. Once done with the changes, you are 
 free to merge to master. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

ElijahTobs committed:
Update README.md

ElijahTobs committed:
fix clone screenshot

ElijahTobs committed:
Merge branch 'project' of https://github.com/ElijahTobs/Newsweek-clone into project

ElijahTobs committed:
Update README.md

- add major features
- add more items to built with
- add getting started

ElijahTobs commented:
@christianotieno 
- I have fixed the screenshot. It's now showing.
- I have also edited the PR message and 
- I have also added instructions on how to clone

ElijahTobs committed:
remove right border from weather (top nav)

ElijahTobs committed:
update screenshot to reflect new changes
